patroni/config.py

```diff
@@ -614,3 +615,25 @@ def get_global_config(self, cluster: Union[Cluster, None]) -> GlobalConfig:
         :returns: :class:`GlobalConfig` object
         """
         return get_global_config(cluster, self._dynamic_configuration)
+
+    def _validate_failover_tags(self):
+        """Checks nofailover/failover_priority config and warns user if it's contradictory"""
+        nofailover_tag = self.get('tags', {}).get('nofailover', None)
```

Comment:
Maybe you could create a temporary variable to reduce copy'n'paste (`self.get('tags', {})`).